e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -204,6 +204,14 @@ describe("scenarios > dashboard > tabs", () => {
       goToTab("Tab 2");
 
       getDashboardCards().should("have.length", cards.length);
+
+      cy.findAllByTestId("toast-undo").should("have.length", cards.length);
+
+      cy.log("undo toasts should be dismiss when saving");
```

Comment:
I added the check here instead of adding a new `it`, I try to do this to not have too many tests which will slow down the CI, and I try to make it explicit what's being tested with a cy.log but the line between "ok" and "too complex" is thin so I'm open to feedback on this

Reply:
I think it makes sense considering the thing that you're testing is basically the same (except `cards.length` vs `0`)